pkgs/games/armagetronad/default.nix

```diff
@@ -129,6 +129,12 @@ let
       nativeBuildInputs = [ autoconf automake gnum4 pkg-config which python3 ]
         ++ extraNativeBuildInputs;
 
+      postInstall = lib.optionalString (!dedicatedServer) ''
+        mkdir -p $out/share/{applications,icons/hicolor}
+        ln -s $out/share/games/armagetronad/desktop/armagetronad.desktop $out/share/applications/
```

Comment:
For compat with 0.4:

`$out/share/games/armagetronad/desktop/armagetronad*.desktop`

Reply:
`nix build '.#armagetronad-dedicated."0.4"'` btw.